schedule.go

```diff
@@ -101,21 +118,37 @@ type PreviewScheduleOptions struct {
 	Overflow bool   `url:"overflow,omitempty"`
 }
 
-// PreviewSchedule previews what an on-call schedule would look like without saving it.
+// PreviewSchedule previews what an on-call schedule would look like without
+// saving it. It's recommended to use PreviewScheduleWithContext instead.
 func (c *Client) PreviewSchedule(s Schedule, o PreviewScheduleOptions) error {
+	return c.PreviewScheduleWithContext(context.Background(), s, o)
+}
+
+// PreviewScheduleWithContext previews what an on-call schedule would look like
+// without saving it.
+func (c *Client) PreviewScheduleWithContext(ctx context.Context, s Schedule, o PreviewScheduleOptions) error {
 	v, err := query.Values(o)
 	if err != nil {
 		return err
 	}
-	var data map[string]Schedule
-	data["schedule"] = s
```

Comment:
These two lines being removed are the bug I am referring to.

Reply:
Nice catch! I have been able to reproduce the bug you mentioned and verified that this approach no longer panics.